tox.ini

```diff
@@ -27,6 +27,11 @@ basepython = python2.7
 deps = -rrequirements/docs.txt
 commands = sphinx-build -b html -d {envtmpdir}/doctrees docs {envtmpdir}/html
 
+[testenv:dennis]
+basepython = python2.7
+deps = dennis
```

Comment:
@jezdez Is there a way I can install a specific git hash here instead of the released version temporarily? The latest has a fix for one string in the it locale that has a URL with `%` character encodings in it. It is a dennis bug that was fixed just yesterday.